test/scripts/e2e_subs/create_destroy_optin_optout.sh

```diff
@@ -23,19 +23,19 @@ ${gcmd} clerk send -a 1000000 -f $ACCOUNT -t $ACCOUNTB
 #############################################
 # Account - create / close / create / close #
 #############################################
-ACCOUNT_NEW=$(${gcmd} account new | grep "Created" | awk '{ sub("\r", "", $NF); print $NF }')
-${gcmd} clerk send -a 100000 -f $ACCOUNT -t $ACCOUNT_NEW
-${gcmd} clerk send -a 0 -f $ACCOUNT_NEW -t $ACCOUNT -c $ACCOUNT
-${gcmd} clerk send -a 100000 -f $ACCOUNT -t $ACCOUNT_NEW
-${gcmd} clerk send -a 0 -f $ACCOUNT_NEW -t $ACCOUNT -c $ACCOUNT
+ACCOUNTC=$(${gcmd} account new | grep "Created" | awk '{ sub("\r", "", $NF); print $NF }')
```

Comment:
It's still weird that this isn't the same form as the `account new` on line 20
Why this form of `grep|awk` vs the above `awk '{ print $6 }'` ?

Reply:
I was fooling around with this trying to get "the last word", I guess I never landed on a consistent formula